webapp/channels/src/components/announcement_bar/text_dismissable_bar.tsx

```diff
@@ -20,18 +20,20 @@ interface Props extends Partial<AnnouncementBarProps> {
     className?: string;
 }
 
-const TextDismissableBar = (props: Props) => {
-    const {allowDismissal, text, ...extraProps} = props;
-    const [dismissed, setDismissed] = useState<boolean>(false);
+const options = {
+    singleline: true,
+    mentionHighlight: false,
+};
+
+const TextDismissableBar = ({allowDismissal, text, onDismissal, ...extraProps}: Props) => {
```

Comment:
Put each prop in a different line.
```suggestion
const TextDismissableBar = ({
    allowDismissal,
    text,
    onDismissal,
    ...extraProps
}: Props) => {
```

Reply:
fixed